src/ontology/mondo-edit.obo

```diff
@@ -156829,10 +156828,10 @@ xref: Orphanet:2073 {source="OMIM:161400"}
 xref: SCTID:46263000 {source="MONDO:equivalentTo"}
 xref: UMLS:C0007384 {source="MONDO:notFoundInDiseaseSubset", source="OMIM:161400"}
 xref: UMLS:C1834372 {source="MONDO:equivalentTo", source="MONDO:ncbi_mim2gene_medline", source="OMIM:161400"}
-is_a: MONDO:0016158 {source="Orphanet:2073/btnt"} ! narcolepsy-cataplexy syndrome
-is_a: MONDO:0021107 {source="MONDO:Redundant", source="OMIM:161400"} ! narcolepsy
-intersection_of: MONDO:0021107 ! narcolepsy
+is_a: MONDO:0021107 {source="MONDO:Redundant", source="OMIM:161400"} ! hereditary narcolepsy
+intersection_of: MONDO:0021107 ! hereditary narcolepsy
 intersection_of: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4847 ! HCRT
+relationship: excluded_subClassOf MONDO:0016158 {source="Orphanet:2073/btnt", source="https://orcid.org/0000-0001-5208-3432"} ! narcolepsy-cataplexy syndrome
```

Comment:
I think this term can have 2 parents: hereditary narcolepsy AND narcolepsy-cataplexy